packages/block-library/src/navigation/editor.scss

```diff
@@ -76,6 +77,13 @@
 	flex-wrap: wrap;
 }
 
+.wp-block-navigation {
+	min-height: 60px;
+	padding-top: 7px;
+	padding-bottom: 7px;
```

Comment:
Magic numbers? Worth checking whether can be created from variables? If not then consider adding comments to explain why they exist.